src/main/scala/io/sdkman/changelogs/TomcatMigration.scala

```diff
@@ -72,4 +72,37 @@ class TomcatMigration {
     ).validate()
       .insert()
       .asCandidateDefault()
+
+  @ChangeSet(
+    order = "010",
+    id = "010-update_tomcat_versions",
+    author = "stefanpenndorf"
+  )
+  def migration010(implicit db: MongoDatabase): Document = {
+    removeVersion("tomcat", "7.0.106")
+    removeVersion("tomcat", "7.0.109")
+    removeVersion("tomcat", "10.0.0-M10")
+    removeVersion("tomcat", "10.0.14")
+    removeVersion("tomcat", "10.0.22")
+    removeVersion("tomcat", "10.1.0-M8")
```

Comment:
You only need to remove the old versions if they are surpassed milestone releases. The old stable versions can stay for historical purposes.

Reply:
I've updated the PR and added the latest releases as well.